packages/core-components/src/components/AlertDisplay/AlertDisplay.tsx

```diff
@@ -29,6 +29,23 @@ import pluralize from 'pluralize';
  * @remarks
  *
  * Shown as SnackBar at the center top of the page by default. Configurable with props.
+ *
+ * @param anchorOrigin.vertical - Vertical orientation of where the AlertDisplay will be located
```

Comment:
@Rugvip, I get warnings in the API Report for these and I'm not sure how to fix them, any suggestions?

Reply:
Not sure tbh, I think you might actually need to document just `anchorOrigin` instead

Reply:
OK, fixed this up by just documenting `anchorOrigin` 👍